packages/next-on-pages/src/buildApplication/processVercelFunctions/configs.ts

```diff
@@ -36,10 +39,14 @@ export async function collectFunctionConfigsRecursively(
 				normalizePath(relative(configs.functionsDir, path)),
 			);
 
-			if (
+			const isPrerenderedIsrFunc =
 				config?.operationType?.toLowerCase() === 'isr' &&
-				!path.endsWith('.action.func')
-			) {
+				!path.endsWith('.action.func');
+			const isPrerenderedApiFunc =
+				config?.operationType?.toLowerCase() === 'api' &&
+				checkPrerenderConfigExists(path, files);
```

Comment:
PS: does this need to be `api` specific? 🤔 

what if we were to just do:
```ts
			const isPrerenderedFunc =
				checkPrerenderConfigExists(path, files);
```
wouldn't that include the `api` ones plus any potential new prerender function?

(or is there the risk of considering prerendered some funcs which shouldn't? 🤔)

Reply:
Well, all ISR functions are prerendered, and this is the first time we're seeing a function type that isn't ISR that is supposed to be treated like ISR. I feel like it's probably a bit safer to stay on the side of caution and only do this for the API ones

Reply:
who knows, maybe one day they'll generate a prerender config for an edge function and we stop doing ssr because we made the check for everything

Reply:
I see 🙂

yeah I think it's ok to keep the `api` check 👍 , it's as you said probably safer and I guess there isn't a real significant benefit right now in skipping it 🤔